tests/data/demo/market/price_scenario_by_id.yaml

```diff
@@ -185,4 +185,5 @@ scenario_26:
   time_series_external_id: '907748'
   transformations:
   - kwargs:
+      value: 1
```

Comment:
This change is what causes tge marketmodel update (update to incremental mapping sequences). I have tested that the same changes occurs when using this config file with the main branch